src/pages/changes.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import Layout from '../components/layout';
+import { StaticQuery, graphql } from 'gatsby';
+import { Document } from '@contentful/rich-text-types';
+import { documentToReactComponents } from '@contentful/rich-text-react-renderer';
+
+import '../styles/changes.scss';
+
+type ChangelogEntry = {
+  title: string,
+  date: string, // e.g. "2019-10-01"
+  body: {
+    json: Document,
+  },
+  node_locale: string,
+};
+
+type YearAndMonth = {year: number, month: number};
+
+/**
+ * Given a date string in the form "YYYY-MM-DD", returns the "YYYY-MM" part.
+ */
+const isSameYearAndMonth = (a: YearAndMonth, b: YearAndMonth) => a.year === b.year && a.month === b.month;
+
+const parseYearAndMonth = (date: string): YearAndMonth => {
+  const year = parseInt(date.slice(0, 4), 10);
+  const month = parseInt(date.slice(5, 7), 10);
+
+  return {year, month};
+};
+
+const MONTHS = {
+  1: 'January',
+  2: 'February',
+  3: 'March',
+  4: 'April',
+  5: 'May',
+  6: 'June',
+  7: 'July',
+  8: 'August',
+  9: 'September',
+  10: 'October',
+  11: 'November',
+  12: 'December',
+};
```

Comment:
Ideally we should rely on the built-in [`Date.prototype.toLocaleString()`](https://stackoverflow.com/a/39972988) for this, but I wasn't sure how supported it is cross-browser so I just hard-coded a list for now.

Reply:
Er, it looks like IE11 actually supports `toLocaleString()` just fine, so I ended up using it!